tools/Makefile.am.inc

```diff
@@ -30,3 +30,19 @@ eos_network_id_CFLAGS = \
 	$(NULL)
 
 eos_network_id_LDADD = $(EOS_INSTRUMENTATION_LIBS)
+
+bin_SCRIPTS = eos-label-location
+
+do_edit = sed \
+	-e "s|\@sysconfdir\@|$(sysconfdir)|"
```

Comment:
Nitpick: I think you want the `g` flag at the end of that. The convention I’ve seen elsewhere for quoting the `@`s is: `-e 's|[@]sysconfdir[@]|$(sysconfdir)|g'`